File: GymApi/GymApi/Models/Usuario.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace GymApi.Models
{
    [Table("Usuarios")]
    public class Usuario
    {
        [Key]
        public int id_usuario { get; set; }

        [ForeignKey("Rol")]
        public int id_rol_fk { get; set; }

        public string nombre { get; set; } = null!;
        public string apellido { get; set; } = null!;
        public string dui { get; set; } = null!;
        public string username { get; set; } = null!;
        public string password { get; set; } = null!;
        public string estado_usuario { get; set; } = "Activo";
        public DateTime fecha_registro { get; set; } = DateTime.Now;

      
        public virtual Rol? Rol { get; set; }
    }
}

File: GymAdmin/src/sv/arrupe/gym/model/Usuario.java
package sv.arrupe.gym.model;

public class Usuario {
    public int id_usuario;
    public String nombre;
    public String apellido;
    public String dui;
    public String estado_usuario;
    public String username;
    public String password;
    public int id_rol_fk;
    
    public void setId_rol_fk(int id_rol_fk) { this.id_rol_fk = id_rol_fk; }
    public void setEstado_usuario(String estado_usuario) { this.estado_usuario = estado_usuario; }
}